.env.production.sample

```diff
@@ -214,3 +214,10 @@ STREAMING_CLUSTER_NUM=1
 # SAML_UID_ATTRIBUTE="urn:oid:0.9.2342.19200300.100.1.1"
 # SAML_ATTRIBUTES_STATEMENTS_VERIFIED=
 # SAML_ATTRIBUTES_STATEMENTS_VERIFIED_EMAIL=
+
+# Use HTTP proxy for outgoing request (optional)
+# http_proxy=http://gateway.local:8118
```

Comment:
Why lowercase name?

Reply:
From the first link:

>But http_proxy and HTTP_PROXY is treated specially under CGI environment.

It seems like neither should be used. Maybe `HTTP_PROXY_URL` instead, since the value is a URL?

Reply:
That's the story behind how http_proxy got used. Since Mastodon does not use CGI, it should not be vulnerable. Also, since the setting of proxy by http_proxy is common, there are also many gems that use it (Try `grep -r http_proxy vendor/`). Therefore, I think that it makes sense to use http_proxy for setting.

Reply:
Okay, http_proxy is fine then